main/logkeeper.go

```diff
@@ -29,15 +30,28 @@ func main() {
 	logPath := flag.String("logpath", "logkeeperapp.log", "path to log file")
 	maxRequestSize := flag.Int("maxRequestSize", 1024*1024*32,
 		"maximum size for a request in bytes, defaults to 32 MB (in bytes)")
-	traceCollectorEndpoint := flag.String("traceCollectorEndpoint", "", "OTEL Collector URL")
-	useInsecure := flag.Bool("useInsecure", false, "Use Internal DNS")
-	sampleRatio := flag.Float64("sampleRatio", 1.0, "Otel Trace Sample ratio")
 	_ = flag.String("dbhost", "", "LEGACY: this option is ignored")
 	flag.Parse()
 
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
+	var (
+		err                    error
+		traceCollectorEndpoint string
+		sampleRatio            float64
+	)
+
+	if value, ok := os.LookupEnv("LK_COLLECTOR_ENDPOINT"); ok {
+		traceCollectorEndpoint = value
+	}
+	if value, ok := os.LookupEnv("LK_TRACE_RATIO"); ok {
```

Comment:
It seems like if `LK_TRACE_RATIO` isn't set in the environment the sample ratio will be 0. Is that what we want? I would have thought we'd want to default to 1.0.